src/components/home/elements/TokenRecord.vue

```diff
@@ -368,16 +367,12 @@ export default {
             const url2 = NODE_IP + '/contract/content/' + contractId
             this.$http.get(url2).then(response => {
                 var tokentype = response.body.textual.descriptors
-                this.functionIndex = -9
                 if (tokentype === CONTRACT_DESCRIPTOR) {
-                    this.functionIndex = SEND_FUNCIDX
                     this.isSplit = false
                 } else if (tokentype === CONTRACT_WITH_SPLIT_DESCRIPTOR) {
-                    this.functionIndex = SEND_FUNCIDX_SPLIT
                     this.isSplit = true
```

Comment:
Line 370 to 376 can simplify as `this.isSplit = tokentype === CONTRACT_WITH_SPLIT_DESCRIPTOR`